cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -356,3 +365,82 @@ func buildDefaultMeshConfig(presetMeshConfigMap *corev1.ConfigMap) (*configv1alp
 
 	return config, util.CreateApplyAnnotation(config, unstructured.UnstructuredJSONScheme)
 }
+
+func (b *bootstrap) ensureMeshRootCertificate() error {
+	listOptions := metav1.ListOptions{
```

Comment:
Similar to the `osm.caBundleSecretName` value option used to create or obtain the ca bundle for Tresor, should the name of the MeshRootCertificate created or obtained on install be optionally set by the user on install? Right now, I am searching for MRCs by their status. If the root cert had been rotated, the name of the active MRC wouldn't be the same as the name of the MRC passed on install.

Reply:
I don't see any reason to provide that configurability since we will only create one if there's no active one. The reason to configure the osm-ca-bundle, was so that a user could pre-populate it. But in this case, if a user pre-populated with an MRC, we would pick that up (or more likely they won't pre-populate an MRC, but they'll prepopulate the underlying secret it will point to)